robot_framework/config.py

```diff
@@ -3,7 +3,25 @@
 # The number of times the robot retries on an error before terminating.
 MAX_RETRY_COUNT = 3
 
+# Whether the robot should be marked as failed if MAX_RETRY_COUNT is reached.
+FAIL_ROBOT_ON_TOO_MANY_ERRORS = True
+
 # Error screenshot config
 SMTP_SERVER = "smtp.aarhuskommune.local"
 SMTP_PORT = 25
 SCREENSHOT_SENDER = "robot@friend.dk"
+
+# Constant/Credential names
+ERROR_EMAIL = "Error Email"
```

Comment:
Is this supposed to be an email address? Is so, state that in the documentation (the comment).

Does it make sense to let `None` be the default value and add a check in the framework for checking that a value has been set? And maybe even that the value is a valid email address?

Reply:
No it's the name of the constant in OpenOrchestrator. In our case it's "Error Email".

This line is also to show a suggestion on how to define which constants the robot uses.

Reply:
I don't understand this. Is the value, i.e. `"Error Email"`, the name of a constant that the robot uses to get a value defined elsewhere? Can I use other values than `"Error Email"`?

Reply:
It's defined in OpenOrchestrator like this:

![image](https://github.com/itk-dev-rpa/Robot-Framework/assets/123645708/b658629a-cc80-480d-9a5f-65f636acd892)

And used in `exceptions.py` like this:

`error_email = orchestrator_connection.get_constant(config.ERROR_EMAIL).value`

Reply:
It enables us to change the value in the future without changing the source code.